### PR TITLE
CN-4873 create hodl landing screen

### PR DESCRIPTION
**Pull request by ShamanShaulic**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/screens/HodlDisabledActions/HodlDisabledActions.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    const { theme, actions, formData } = this.props;
+
+    const checkboxCopy = [
+      { field: 1, explanation: "You will not have ability to Withdraw funds" },
+      { field: 2, explanation: "You will not have ability to CelPay funds" },
+      {
+        field: 3,
+        explanation:
+          "You will not have ability to change whitelisted withdrawal addresses",
+      },
```
Comment (fjovakaric) [inline app/components/screens/HodlDisabledActions/HodlDisabledActions.js]: name fields `hodl1`, `hodl2`, `hodl3`

File: app/components/screens/HodlDisabledActions/HodlDisabledActions.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+              }
+              margin={"20 0 20 0"}
+              padding={"15 15 0 15"}
+            >
+              <CelCheckbox
+                onChange={(field, value) =>
+                  actions.updateFormField(field, value)
+                }
+                field={`hodl${checkboxCopy[i].field}`}
+                value={formData[`hodl${checkboxCopy[i].field}`]}
```
Comment (fjovakaric) [inline app/components/screens/HodlDisabledActions/HodlDisabledActions.js]: should remove `hodl`

File: app/components/screens/HodlDisabledActions/HodlDisabledActions.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+import CelButton from "../../atoms/CelButton/CelButton";
+
+@connect(
+  state => ({
+    theme: state.user.appSettings.theme,
+    formData: state.forms.formData,
+  }),
+  dispatch => ({ actions: bindActionCreators(appActions, dispatch) })
+)
+class HodlDisabledActions extends Component {
```
Comment (fjovakaric) [inline app/components/screens/HodlDisabledActions/HodlDisabledActions.js]: Rename screen and file to HODLInfoCheckboxes

File: app/navigator/Navigator.js [hunk 4]
```diff
@@ -242,6 +243,7 @@ export const screens = {
   ChangePin,
   TwoFactorSettings,
   TwoFaAuthAppConfirmationCode,
+  HodlDisabledActions,
```
Comment (fjovakaric) [inline app/navigator/Navigator.js]: HODLLanding screen?

File: app/components/screens/HODLInfoCheckboxes/HODLInfoCheckboxes.styles.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+const themed = {
+  light: {},
+
+  dark: {},
+
+  celsius: {},
+};
+
+const HodlDisabledActionsStyle = () => getThemedStyle(base, themed);
```
Comment (fjovakaric) [inline app/components/screens/HODLInfoCheckboxes/HODLInfoCheckboxes.styles.js]: HODLInfoCheckboxes
